### DISCUSSION TITLE
q2_2020_87

### DISCUSSION THREAD

**Discussion by shay-bi** [Koren-Ben-Ezra/Exams#10] [Bug Discussions🪲]
אני אשמח להסבר מדוע on ולא on בריבוע

**Comment by korenbenezra** [comment]
הסיבוכיות O(n^2)
תוקן, תודה.
* נבצע לכל היותר n^2/2 איטרציות בסיבוכיות זמן קבועה לכן הסיבוכיות זמן O(n^2)

**Comment by shay-bi** [comment]
בכיף אחSent from my iPhoneOn 29 Jun 2023, at 22:59, Koren Ben Ezra ***@***.***> wrote:﻿
הסיבוכיות O(n^2)
תוקן, תודה.

נבצע לכל היותר n^2/2 איטרציות בסיבוכיות זמן קבועה לכן הסיבוכיות זמן O(n^2)
